rerun_py/src/python_bridge.rs

```diff
@@ -951,8 +956,9 @@ fn log_annotation_context(
         [annotation_context].as_slice(),
     );
 
-    let msg = (&row.into_table())
-        .try_into()
+    let msg = row
+        .into_table()
+        .as_arrow_msg()
         .map_err(|err: DataTableError| PyValueError::new_err(err.to_string()))?;
```

Comment:
seems like a small helper here would go a long way

```suggestion
    let msg = row
        .to_arrow_msg()
        .map_err(|err: DataTableError| PyValueError::new_err(err.to_string()))?;
```